backend/src/main/resources/db/migration/V1.25.0__NOTIF-291_event_cleaning_warm_up.sql

```diff
@@ -0,0 +1,5 @@
+-- This will ease the first run of EventLogCleaner by reducing the amount of data it will have to delete.
+-- The following date is based on the assumption that the event log retention delay is 31 days.
+
+DELETE FROM event
+WHERE created < '2021-07-31T00:00:00.000000';
```

Comment:
This date has to be updated as late as possible, ideally right before we deploy on prod.

Reply:
I changed the value to `now() minus 28 days` which means `EventLogCleaner` will start with the lowest possible number of records to delete on its first run, assuming we deploy notifications-backend on prod before thursday.

The retention delay should be 31 days and we will actually have 28 days of data but it doesn't matter because that data isn't used yet in the prod UI.